activity-tracking/features/exercise.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Exercise Management
+
+   Scenario: Add an exercise
+    Given I have no exercises
+    When I add an exercise
+    Then the exercise should be added to my list
+
+  Scenario: Get all exercises
+    Given I have an exercise
+    When I get all exercises
+    Then I should see my exercise in the list
+
+  Scenario: Cannot add an exercise with the same name
+    Given I have added an exercise named "Running"
+    When I try to add another exercise named "Running"
+    Then I should see an error message that the exercise already exists
```

Comment:
We should be able to add the same exercise again, right?

Reply:
Yeah, this case will not be relevant. I have removed the ones not relevant.